internal/apiserver/config/defaults.go

```diff
@@ -80,6 +81,7 @@ var prodConfig = Configuration{
 		JtiExpiration: 15 * time.Minute,
 	},
 	LocalStorageBasePath: "/data",
+	ValidationServerUrl:  "https://dev-einvoice-validator.mypaas.vnet.sk",
```

Comment:
delete this, this service will not have public IP (now it has only for testing purposes) and we will set local IP in env variable